src/Input/inputStyles.js

```diff
@@ -16,6 +16,17 @@ const invalidStyles = (theme) => css`
   background: ${theme.colors.palette.red.lighter};
 `;
 
+/* stylelint-disable property-no-vendor-prefix */
```

Comment:
Added these because of stylelint didn't like the vendor prefixes.

I personally think it's a good idea to minimize the effect of these styles.

I didn't see any problems with removing the prefixes, however I'm concerned about potential browser changes in the future that could break things.